automatic/php/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,7 @@
+﻿$packageName = 'php'
+
+$installLocation = Join-Path $(Get-ToolsPath) $packageName
+Write-Host "Deleting $installLocation"
+rm -Force -Recurse $installLocation
+
+#Uninstall-ChocolateyPath $installLocation
```

Comment:
why is this commented out?
shouldn't the install location be removed from path environment variable?